app/page.tsx

```diff
@@ -79,17 +83,17 @@ export default async function Page({ searchParams }: { searchParams: { page: str
         // Since the resulting image will be a square, we don't really need to fetch the metadata
         // but let's do it anyway to show how it's done. Meta data provides both the original
         // and transformed dimensions of the image.
-        const metadata = await fetchMetadata(metadataUrl);
+        // const metadata = await fetchMetadata(metadataUrl);
 
-        if (!url || !metadata) {
+        if (!url) {
           return undefined;
         }
 
         const thumb = {
           url,
           attributes: {
-            width: metadata.width, // Post transform width
-            height: metadata.height // Post transform height
+            width: IMAGE_SIZE, // Post transform width
+            height: IMAGE_SIZE // Post transform height
```

Comment:
I know this saves the extra fetch, but it removes an important part of learning how to work with Xata's image transformation tools. I'd prefer we keep the meta data stuff in here, because if they change the transform to a 'cover' or 'contain' or something this breaks.

I'm less worried about this repo being blazing fast, and care more about it being something we can point to as an example of how to do certain things.

Dunno.

Reply:
As discussed, I updated the comment with why it's not needed.